src/Form/Form.js

```diff
@@ -73,8 +70,8 @@ class Form {
         }
 
         this.getValues = () => {
-            const username = [...this.emailInputs][0]?.value
-            const password = [...this.passwordInputs][0]?.value
+            const username = [...this.emailInputs].reduce((prev, curr) => curr.value ? curr.value : prev, '')
+            const password = [...this.passwordInputs].reduce((prev, curr) => curr.value ? curr.value : prev, '')
```

Comment:
Some pages keep empty fields with the stated purpose (explicitly mentioned in a comment) of fooling password managers. Because users must suffer 🤦. This scans the whole form.